src/test/resources/com/puppycrawl/tools/checkstyle/javadocpropertiesgenerator/InputJavadocPropertiesGeneratorCorrect2.java

```diff
@@ -0,0 +1,10 @@
+package com.puppycrawl.tools.checkstyle.javadocpropertiesgenerator;
+
+public class InputJavadocPropertiesGeneratorCorrect2 {
+
+    // used here to avoid confusing the javadoc tool
+    /**
+     * An annotation of a package, type, field, parameter or variable.
```

Comment:
Put here some words to explain reason of this test

Reply:
done